docs/architecture/baking.md

```diff
@@ -19,33 +19,79 @@ The baking process includes many steps that Tezos users and developers don't nee
+1. If the first baker doesn't publish a block in time, the next baker on the list has a certain amount of time to bake a block (a new round), and so on until a block is created.
 1. The validators verify the block and publish their attestations in future blocks.
 1. The next list of bakers create the next block, until the end of the cycle.
 
+Note that:
+- The presentation above is somewhat simplified; in reality, validation is done in two phases, called pre-attestation and attestation.
+- For any given block and round, only one baker has the right to create the block, and several bakers have the right to (pre-)attest the block.
+
 ## Becoming a baker
 
-To become a baker, you must create an account to act as a _delegate_, which is an account that is authorized to bake blocks and attest blocks that other accounts bake.
+To become a baker, you must create an account to act as a _delegate_, which is an account that is authorized to bake blocks and attest blocks that other accounts bake, and also to receive delegations (and stake) from other accounts, as explained later.
 Delegates temporarily stake tez as a security deposit to ensure that they are acting according to the rules of the protocol.
-This stake stays locked for a certain number of cycles before the delegate can begin baking.
-Similarly, when bakers stop baking, their stake is unlocked after a certain number of cycles.
+Deposits can be "unstaked" later, either partially (at any time) or totally (when bakers stop baking).
+There is a delay of a certain number of cycles from the moment when the tez are staked until the delegate can begin baking.
+Similarly, when tez are unstaked, they are unlocked after a certain number of cycles.
+Staked tez may be slashed by the protocol if the baker misbehaves (e.g., proposes or attests two different blocks for the same level).
+
+A delegate participates in consensus in proportion to their _baking power_: the more baking power a delegate has, the more likely it is to be selected to bake or to validate blocks and thus receive the rewards.
+The baking power of a delegate is computed from the amounts of tez staked (by its own and by all its stakers) and owned (by its own and by all its delegators), knowing that non-staked tez are weighted half as much as staked tez in the sum.
 
-The delegate must stake at least least 6,000 tez, either from its own account or from tez that delegators delegate to it.
-The more tez a delegate has, both from its own account and from delegators, the more likely it is to be selected to bake a block and thus receive the rewards.
+The delegate must have a baking power of at least 6,000 tez to be allowed to bake.
 
-Bakers must run at least one Tezos node and a baker daemon to go with it.
-These daemons must run at all times with a stable internet connection, because inactive bakers are automatically removed from the network.
+A delegate also participates in [governance](./governance) in proportion to their _voting power_.
```

Comment:
Use root-relative links to avoid a Docusaurus routing problem:
```suggestion
A delegate also participates in [governance](/architecture/governance) in proportion to their _voting power_.
```